database/management/commands/load_fixtures.py

```diff
@@ -13,7 +13,8 @@ class Command(BaseCommand):
     Loads necessary fixtures
     """
 
-    fixtures = ["base_schedule.json", "departments.json", "teacher.json"]
+    fixtures = ["base_schedule.json", "departments.json"
+                ]
```

Comment:
why was this changed? seems unintentional

Reply:
must have been a mistake my apologies